pkgs/development/php-packages/grpc/default.nix

```diff
@@ -1,19 +1,38 @@
-{ buildPecl
-, zlib
+{ stdenv
+, fetchFromGitHub
+, autoreconfHook
+, pkg-config
+, php
 , lib
+, grpc
 }:
 
-buildPecl {
+stdenv.mkDerivation rec {
```

Comment:
Wouldn't it be a good idea to keep `buildPecl` and override the `src` attribute?

Reply:
Ahh yea, we can still use it :)